plugins/scaffolder-backend/src/scaffolder/tasks/DatabaseTaskStore.ts

```diff
@@ -596,10 +653,22 @@ export class DatabaseTaskStore implements TaskStore {
   ): Promise<void> {
     const { taskId, body } = options;
     const serializedBody = JSON.stringify(body);
-    await this.db<RawDbTaskEventRow>('task_events').insert({
-      task_id: taskId,
-      event_type: 'cancelled',
-      body: serializedBody,
+    const [ret] = await this.db<RawDbTaskEventRow>('task_events')
+      .insert({
+        task_id: taskId,
+        event_type: 'cancelled',
+        body: serializedBody,
+      })
+      .returning('id');
+
+    this.events?.publish({
+      topic: 'scaffolder.task.event',
```

Comment:
I wonder if these topic names could either be aligned or rethought.

It feels a little strange that we have `scaffolder.task.event` and `scaffolder.task` as topics, but I think that the `.event` suffix is a little odd, but understand it's because of things on the event stream.

Do you have any other suggestions here?

Reply:
I don't have any other suggestions really. These are scaffolder task events :D

Reply:
So should the topic just be `scaffolder.task` for all of them or no?

Reply:
I think it's good idea to separate these. Some might not be interested in all events but just the state changes of the task. WDYT?

Reply:
@drodil can we filter out logs here? As this topic otherwise can be very chatty.
Right now how is interested can already subscribe to log stream.

I think the confusion here that `.event` is not about saying that it is events, but that it is saved in the `db` as task_events.

Reply:
Should be refactored, WDYT?